inc/functions_upload.php

```diff
@@ -313,6 +313,11 @@ function upload_avatar($avatar=array(), $uid=0)
 		case "image/x-png":
 			$img_type = 3;
 			break;
+		case "image/bmp":
+		case "image/x-bmp":
+		case "image/x-bitmap":
+			$img_type = 4;
```

Comment:
@Destroy666x I haven't tested it (yet) but `IMG_BMP` not mentioned in the documentation: http://php.net/manual/en/image.constants.php